ui/webui/resources/fonts/poppins.css

```diff
@@ -430,3 +430,148 @@
     src: url(./third_party/poppins/Poppins_900_normal_bfa3a4d70cc392ea9c1260963baae6ed_latin_v5.woff2) format('woff2');
     unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2212, U+2215;
 }
+
+/* manrope-100 */
```

Comment:
nit: comment as to why including manrope here (i.e. what the unicode ranges are)